model_prices_and_context_window.json

```diff
@@ -981,35 +981,45 @@
         "litellm_provider": "gemini",
         "mode": "chat"
     },
-    "command-nightly": {
+
+    "cohere_chat/command-r": {
+        "max_tokens": 128000, 
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.00000050,
+        "output_cost_per_token": 0.0000015,
+        "litellm_provider": "cohere_chat",
+        "mode": "chat"
```

Comment:
It seems to support Tool Usage: https://docs.cohere.com/docs/tool-use#step-1

Can we enable function calling here?